.github/CODEOWNERS

```diff
@@ -7,6 +7,14 @@
 /monitoring/prometheus                          @ceph/dashboard
 /doc/mgr/dashboard.rst                          @ceph/dashboard
 
+# Dashboard API team
+/src/pybind/mgr/dashboard/controllers                  @ceph/api
+/src/pybind/mgr/dashboard/frontend/src/app/shared/api  @ceph/api
+/src/pybind/mgr/dashboard/run-backend-api-tests.sh     @ceph/api
+/qa/tasks/mgr/dashboard                                @ceph/api
+/qa/tasks/mgr/test_module_selftest.py                  @ceph/api
+/qa/tasks/mgr/test_dashboard.py                        @ceph/api
+
```

Comment:
Thank you for adding these!

I'm wondering if `/qa/suites/rados/dashboard` or `/qa/suites/rados/dashboard/tasks/dashboard.yaml` (I'm not really sure which of these two makes more sense) can be added here as well.

Reply:
Thanks! Updated with the first suggestion.

Reply:
Thank you!